packages/common/package.json

```diff
@@ -0,0 +1,55 @@
+{
+    "name": "@concordium/common",
+    "version": "0.0.1",
```

Comment:
I suggest that we start at `1.0.0` as everything in this package has already been published before as part of node-sdk. Or at the very least `0.1.0` :smile:

Reply:
I'll set it to 1.0.0 :+1: